src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1356,6 +1375,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                              (shift ? Settings::KeyModifiers::Shift : Settings::KeyModifiers::None) };
     }
 
+    bool TermControl::CloseOnExit() const noexcept

```

Comment:
Some methods in this file appear to have doc comments and others do not.

Reply:
Especially because I'm not totally clear on what the difference is between `Close()` and `CloseOnExit()` just by the names.

Reply:
Oh, is `CloseOnExit()` not a verb? It's a property? Maybe `IsCloseOnExit()` or `ShouldCloseOnExit()` then?